README.md

```diff
@@ -1,2 +1,104 @@
-# resource_library
-Developed by Coding Ninjas
+# Resource Library
+> Connecting students to instructors.   
+
+## Table of contents
+* [General info](#general-info)
+* [Project Demo](#project-demo)
+* [Project Video](#project-video)
+* [Technologies](#technologies)
+* [Setup](#setup)
+* [Features](#features)
+* [Inspiration](#inspiration)
+* [Contact](#contact)
+* [License](#license)
+
+## General info
+Resource Library aims to connect students to instructors who are knowledgeable in a variety of subjects. 
+
+<div align="center">Welcome to Resource Library </div>
+<br/>
+<div align="center">
+<kbd>
+<img src="">
+</kbd>
+</div>
+
+<br/>
+<div align="center">
+<kbd>
+<img src="">
+</kbd>
+</div>
+
+## Project Demo 
+[Click to view site]()
+
+## Project Video
+[Click to view demo of BugView]()
+
+## Technologies
+### Backend Development 
+* Node.js - version 
+* Express - version 
+* Express-Validator - version 
+* Config - version 
+* JWT - version 
+* Mongoose - version 
+* Request - version 
+
+### Frontend Development 
+* JavaScript (ES6)
+* HTML5
+* CSS3
+* React.js - version 
+* React-DOM - version 
+* React-Router-DOM - version 
+* React-Materialize - version 
+* React-Scripts - version 
+* Chart.js - version 
+* React-Chartjs-2 - version 
+* Material-UI - version 
+* Materialize-CSS - version 
+
+## Setup
+To use this project: 
+1. Clone the GitHub repository locally to your computer
+1. In the command line, navigate to the root directory of the repository, and type the following: 
+  $ npm install 
+1. Navigate to the client folder, and in the root directory of the client folder, type the following: 
+  $ npm install 
+1. In the client folder, and in the root directory of the client folder, type the following: 
+  $ npm start
+1. Navigate back to the root directory of this project "/GitConnect" and start the server by typing the following: 
+  $ npx nodemon server 
+
+## Code Examples
+### Node.js/Express.js
+
+### JavaScript/React.js 
+
+
+
+## Features
+* Full stack web application utilizing the MERN stack: MongoDB, Express.js, React, and Node.js. 
+* Authorization and authenication implemented with JWT and bcrypt. 
+* Front-End styles and chart components built with Materialize and Chart.js. 
```

Comment:
I believe we are using Chakra UI instead of Materialize and Chart.js. Will you please make the changes?